ludwig/schema/__init__.py

```diff
@@ -59,6 +59,12 @@ def validate_config(config):
     def custom_is_array(checker, instance):
         return isinstance(instance, list) or isinstance(instance, tuple)
 
+    # Update config from previous versions to check that backwards compatibility will enable a valid config
+    from ludwig.utils.backward_compatibility import upgrade_to_latest_version
+
+    config["ludwig_version"] = "0.4"
```

Comment:
Will this break if the config is in fact not from 0.4? As in either newer or older?

Reply:
No it shouldn't, but I am actually adding a quick update to that section which basically checks if the ludwig version is already set so we don't have to overwrite it.

Reply:
But worst case scenario, if the version is 0.4, all the back compat updates are ran and they have no affect if the config is already valid with 0.6

Reply:
This may be un-needed if we merge Travis's change here: https://github.com/ludwig-ai/ludwig/pull/2424